NewHaven/src/Resources.cpp

```diff
@@ -111,27 +159,20 @@ HarvestDeck::~HarvestDeck()
 	delete[] this->deckContent;
 }
 
+std::uint_least8_t HarvestDeck::getDeckSize()
+{
+	return *this->deckSize;
+}
+
 HarvestTile* HarvestDeck::draw()
```

Comment:
should draw() be renamed DrawHarvestTile() as per the assignment instructions?

Reply:
The instructions are a bit ambiguous, but from what I understood of them is that the function in the deck class is simply draw(), DrawHarvesTile() will be a function in the player class utilizing draw() from the deck.

Reply:
Gotcha